File: 03checkpalindrome/checkpalindrome.py
def solution(inputString):
    reverse = ""
    for c in inputString:
        reverse = c + reverse
    
    if reverse == inputString:
        return True
    else:
        return False

if solution(input("Enter a word, I will check if it is palindrome: ")):
    print("This word is palindrome.")
else:
    print("This word is not palindrome")

File: 03checkpalindrome/checkpalindrome.cpp
#include <iostream>
using namespace std;

bool solution(string inputString){
	string reverseInput = string(inputString.rbegin(), inputString.rend());
	return (inputString == reverseInput) ? true : false;
}
int main(void){
	cout << "Enter word to check if it is palindrome" << endl;
	string word;
	cin >> word;
	cout << solution(word);
	
	cout << "####################" << endl;

	string texto = "Erace una vez un barco chiquito" ;

	cout << "Texto : " << texto << endl;
	//cout << "Rbegin: " << texto.rbegin() << endl;
	//cout << "Rend  : " << texto.rend() << endl;

	return 0;
}
